elements/ipa-extra-hardware-ram/finalise.d/99-ddr-timings-cleanup

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if [ "${DIB_DEBUG_TRACE:-0}" -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+install-packages -e gcc
```

Comment:
You can uninstall make as well, although I expect that's trivial in size compared to gcc.

Reply:
mainly basing this off the ironic-agent element which brings in make as a transitive dependency and doesn't uninstall it:

https://github.com/openstack/diskimage-builder/blob/168127b60ac314c7226bb7e8bee758cb7911de59/diskimage_builder/elements/ironic-agent/finalise.d/99-remove-extra-packages#L19

probably best to build a package so that you only need the run time dependencies, but that could be improved at a later point in time.